package/MDAnalysis/core/topologyattrs.py

```diff
@@ -1286,14 +1293,15 @@ def radius_of_gyration(group, pbc=False, **kwargs):
         ('radius_of_gyration', radius_of_gyration))
 
     @warn_if_not_unique
-    def shape_parameter(group, pbc=False, **kwargs):
+    @_pbc_to_wrap
+    def shape_parameter(group, wrap=False, **kwargs):
```

Comment:
Can we get rid of kwargs here? They are not used.

Reply:
Should this function also be able to use `unwrap`? (If yes, it can be a new issue.)